conans/client/grapher.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans.model.ref import ConanFileReference
+
+class ConanGrapher(object):
+    def __init__(self, project_reference, deps_graph):
+        self._deps_graph = deps_graph
+        self._project_reference = project_reference
+
+    def graph(self):
```

Comment:
I agree with you that it makes sense for `--graph` to print to stdout; that way someone could pipe it into another program if they wanted to. It just means that users will need to supply the conanfile location before the options which is fine with me (`conan info .. --graph`).

I'll modify the tests to reflect the changes and push them up.